controller.rst

```diff
@@ -116,19 +116,30 @@ For more information on routing, see :doc:`/routing`.
 .. index::
    single: Controller; Base controller class
 
-The Base Controller Class & Services
-------------------------------------
+.. _anchor-name:
+   :ref:`The Base Controller Classes & Services <the-base-controller-class-services>`
```

Comment:
I'm not familiar with this `anchor-name` thing - were you trying to make sure the old anchor still worked? If so, #7867 for an example. I'll update this in my PR